app/hooks/why_did_you_update.ts

```diff
@@ -30,7 +32,7 @@ export function useWhyDidYouUpdate(name: string, props: any) {
             // If changesObj not empty then output to console
             if (Object.keys(changesObj).length) {
                 // eslint-disable-next-line no-console
-                console.log('[why-did-you-update]', name, changesObj);
+                logInfo('[why-did-you-update]', name, changesObj);
```

Comment:
This was intended to be left as a console.log, since it shouldn't get to production, so 0/5 on whether we should have it as console.log or a log function.

But if we go with the log function, for sure is logDebug and you should remove the eslint disable comment.